distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -89,6 +89,9 @@ type Decimal
     ## PRIVATE
     Value (big_decimal : BigDecimal)
```

Comment:
Use `private` keyword.

Reply:
Done

Reply:
I see `private` keyword is gone. Why? 1381d29632a0734cd5b8fcaaaff1f12947e32560 doesn't explain why that has happened.

Reply:
When I put the `private` keyword back, I get this:

Private access error: The project-private method 'null.big_decimal' in unknown project cannot be accessed from project 'Standard.Base'.

This is happening **in a method of Decimal**, which should have access. I don't know why it calls it an "unknown project".

Full test failure:
```
    - [FAILED] should be able to construct a Decimal with dec [43ms]
        Reason: An unexpected panic was thrown: (Private_Access.Error 'Standard.Base' Nothing 'null.big_decimal')
        at <enso> Decimal.unscaled_value<arg-0>(/Users/gmt/dev/enso/enso/built-distribution/enso-engine-0.0.0-dev-macos-aarch64/enso-0.0.0-dev/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso:1169:38-53)
        at <enso> Decimal.unscaled_value(/Users/gmt/dev/enso/enso/built-distribution/enso-engine-0.0.0-dev-macos-aarch64/enso-0.0.0-dev/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso:1169:38-67)
        at <enso> Decimal.internal_representation(/Users/gmt/dev/enso/enso/built-distribution/enso-engine-0.0.0-dev-macos-aarch64/enso-0.0.0-dev/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso:1173:55-73)
        at <enso> Decimal.should_have_rep<arg-0>(/Users/gmt/dev/enso/enso/test/Base_Tests/src/Data/Decimal_Spec.enso:14:36-63)
        at <enso> Decimal.should_have_rep(/Users/gmt/dev/enso/enso/test/Base_Tests/src/Data/Decimal_Spec.enso:14:36-82)
```

Reply:
#10799